packages/rainbowkit/src/components/RainbowButton/RainbowButton.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { touchableStyles } from '../../css/touchableStyles';
+import { useWalletConnectors } from '../../wallets/useWalletConnectors';
+import { AsyncImage } from '../AsyncImage/AsyncImage';
+import { Box } from '../Box/Box';
+import * as styles from './RainbowButton.css';
+import { WalletButtonRenderer } from './WalletButtonRenderer';
+
+export const RainbowButton = () => {
+  const [isMouseOver, setIsMouseOver] = useState<boolean>(false);
```

Comment:
Let's keep this private API as `WalletButton` so that the user could use it for MetaMask and other connectors too